src/test/java/bio/terra/aws/resource/discovery/S3EnvironmentDiscoveryTest.java

```diff
@@ -33,6 +34,7 @@ public S3EnvironmentDiscoveryTest() throws URISyntaxException {
 
     s3Client =
         S3Client.builder()
```

Comment:
I had a default region set on my system so this worked; GHA does not so it throws without specifying a region at build time.